src/platform/forms-system/src/js/web-component-patterns/fileInputPattern.jsx

```diff
@@ -134,6 +134,7 @@ export const fileInputSchema = {
     },
     warnings: {
       type: 'array',
+      'ui:hidden': true,
```

Comment:
Without this, the warnings array is made to be an editable array field on the review page. Weird.